docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -29,7 +30,7 @@ For more information about each preview release, see the following announcements
 
 ## Production supported preview
 
-.NET Core Preview 7 is considered production ready by Microsoft and is fully supported. Starting with preview 7, releases will focus on polishing .NET Core 3.0 instead of adding new features. For more information about what has changed in preview 7, see the [preview 7 announcement](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-0-preview-7/).
+.NET Core Preview 8 is considered production ready by Microsoft and is fully supported. Starting with preview 7, releases will focus on polishing .NET Core 3.0 instead of adding new features. For more information about what has changed in preview 8, see the [preview 8 announcement](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-0-preview-8/).
```

Comment:
I think it would be important mention this that is also mentioned in the blog post

"If you are using .NET Core 3.0 Preview 7, you need to move to Preview 8 for “Go Live” support."